dynamodb/src/main/scala/akka/stream/alpakka/dynamodb/javadsl/DynamoClient.scala

```diff
@@ -49,6 +52,8 @@ final class DynamoClient(settings: DynamoSettings)(implicit system: ActorSystem,
 
   def query(request: QueryRequest) = single(Query(request))
 
+  def queryAll(request: QueryRequest) = source(Query(request))
```

Comment:
Could you add the return type on this, please. (and while at it to the other methods, as well?)